docs/guides/soloproject/topics/tier_developer_python.md

```diff
@@ -0,0 +1,95 @@
+# Developer Tiers ( Non Web Developer - Python )
```

Comment:
Suggest either adding the word `Tiers` to other role topic headings or remove it here for consistency